.ci/static-checks.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/bash
 #
-# Copyright (c) 2018 Intel Corporation
+# Copyright (c) 2018,2020 Intel Corporation
 #
 # SPDX-License-Identifier: Apache-2.0
-#
 
-set -o errexit
-set -o nounset
-set -o pipefail
+set -e
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
```

Comment:
seems like we need to add the `lib.sh` in the `.ci` directory

Reply:
doh, thanks. I forgot to `git add` it .... I did add and test locally....